### ISSUE TITLE
mention/link etherpad generator in instructor notes

### ISSUE THREAD

**Issue by brownsarahm** [carpentries/instructor-training#1126]
We should edit [here](https://github.com/carpentries/instructor-training/blob/gh-pages/_extras/guide.md) to add the generator or replace the template etherpad with the generator that's under extras. 

I think it's worth some discussion of if we want to replace the etherpad template (that way it's less maintenance) or keep it (advantages tbd?)

**Comment by karenword** [comment]
In the long run, I would like to use the generator in place of the old template. For now, I find that I prefer to use the original etherpad template vs using the generator. I don't mind updating it when I use it. It would be good to know what others are doing, though!

In the next update, I think there are things we can do to optimize the curriculum around the generator so that we can finally retire the old template.

**Comment by Denubis** [comment]
Let me know what changes you need? Easy enough to fork or change.

**Comment by ChristinaLK** [comment]
I always build up my etherpad manually b/c I don't like pre-populating the whole thing. :joy:

**Comment by karenword** [comment]
@Denubis to be clear, I think the etherpad generator is fantastic. What I was thinking we could do is engineer the curriculum around its selection criteria such that exactly the text we want in the etherpad is in the right places to be picked up by the generator. :) At that point there may be tweaks that become clear, but until then I can't think of a good change!

**Comment by brownsarahm** [comment]
Christina's comment reminded me that I actually just set up for my upcoming workshop in the way I do *not* prefer. I used the generator and pre-filled the etherpad, what I actually prefer (and did 2/3 times I taught previously) is a separate YYYY-MM-DD-LOC-prep etherpad for the preparation that I do and manually copy things over (sometimes with additinoal on the fly customization)  during the workshop.  I also arranged it in those two times so that I did the copying mostly for my co-instructor and they did it while I was teaching.
